iohk/iohk-ops.hs

```diff
@@ -313,9 +311,6 @@ runNew o@Options{..} New{..} args = do
   echo $ "-- " <> (unsafeTextToLine $ configFilename) <> " is:"
   cmd o "cat" [configFilename]
 
```

Comment:
Create should still exist here, to avoid extra steps when creating a new deployment